src/map/Node.kt

```diff
@@ -9,11 +12,16 @@ class Actor(private val lifeUnits: Int) : MapObject {
 
 }
 
+class AlwaysGoUpPlayer(override val actor: Actor): Player {
```

Comment:
why is this specific test implementation in production sources?

Reply:
my mistake, I wanted to use it for conditional exception handling - but we already discussed that didn't make sense